kubernetes/configs/qa/agreement_email_sender.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+INTEROP_FE_BASE_URL="selfcare.refactor.dev.interop.pagopa.it"
```

Comment:
```suggestion
INTEROP_FE_BASE_URL="selfcare.qa.interop.pagopa.it"
```
Not a big deal since it's mocked 😄